modular_skyrat/modules/fishing/code/fishing.dm

```diff
@@ -71,28 +77,53 @@ GLOBAL_LIST_INIT(fishing_weights, list(
 		QDEL_NULL(mutate_parent)
 	if(!fisher)
 		return
+	var/obj/item/fishing_rod/fisher_rod = fisher
```

Comment:
couldn't we just typecast `fisher` to be `obj/item/fishing_rod` in the `finish_fishing()` proc? I don't think it'd be anything else